cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -200,7 +207,36 @@ friendlyTxBodyImpl
                   , "update proposal" .= friendlyUpdateProposal txUpdateProposal
                   , "validity range" .= friendlyValidityRange era (txValidityLowerBound, txValidityUpperBound)
                   , "withdrawals" .= friendlyWithdrawals txWithdrawals
-                  ])
+                  ]) ++
+                (inEonForEra
+                  []
+                  (\(cOnwards :: ConwayEraOnwards era) ->
+                      case txProposalProcedures of
+                       Nothing -> []
+                       Just (Featured _ TxProposalProceduresNone) -> []
+                       Just (Featured _ (TxProposalProcedures lProposals _witnesses)) ->
+                         ["governance actions" .= (friendlyLedgerProposals cOnwards $ toList lProposals)])
+                      era)
+                    ++
+                (inEonForEra
+                  []
+                  (\cOnwards ->
+                    case txVotingProcedures of
+                       Nothing -> []
+                       Just (Featured _ TxVotingProceduresNone) -> []
+                       Just (Featured _ (TxVotingProcedures votes _witnesses)) ->
+                             ["voters" .= (friendlyVotingProcedures cOnwards votes)])
+                  era)
+  where
+    friendlyLedgerProposals :: ConwayEraOnwards era -> [L.ProposalProcedure (ShelleyLedgerEra era)] -> Aeson.Value
+    friendlyLedgerProposals cOnwards proposalProcedures =
+      Array $ Vector.fromList $ map (friendlyLedgerProposal cOnwards) proposalProcedures
+
+friendlyLedgerProposal :: ConwayEraOnwards era -> L.ProposalProcedure (ShelleyLedgerEra era) -> Aeson.Value
+friendlyLedgerProposal cOnwards proposalProcedure = object $ friendlyProposalImpl cOnwards (Proposal proposalProcedure)
+
+friendlyVotingProcedures :: ConwayEraOnwards era -> L.VotingProcedures (ShelleyLedgerEra era) -> Aeson.Value
+friendlyVotingProcedures cOnwards x = conwayEraOnwardsConstraints cOnwards $ toJSON x
```

Comment:
@carbolymer, @Jimbo4350> defaulting to the ledger's JSON instance here. Can you confirm that's what we want?

TODO @smelc Inline this if this is what we want

Reply:
This is fine. Would pretty json be nicer to read?

Reply:
@Jimbo4350> I'm not sure what you mean, since we're returning `Aeson.Value` here. Prettying (if any) the String output happens in the caller and I didn't change it.

So I'm leaving as is, but let me know later if you want to revisit.